src/DropDownMenu/DropDownMenu.js

```diff
@@ -84,6 +81,14 @@ class DropDownMenu extends Component {
   // than just the parent.
   static propTypes = {
     /**
+     * This is the point on the anchor where the popover's
+     * `targetOrigin` will attach to.
```

Comment:
Either: "This is the point on the anchor where the popover's `targetOrigin` will attach."

Or: "This is the point on the anchor that the popover's`targetOrigin` will attach to.